### ISSUE TITLE
Pre-trained models

### ISSUE THREAD

**Issue by DavidHilly** [facebookresearch/svoice#31]
When will the Pre-trained models be available?

Thanks

**Comment by adamfils** [comment]
I trained the model on the toy dataset for about 500 Epochs
I could give you the link to my model if you want
It does pretty good on some files
Will have to train on a larger dataset.

**Comment by egorsmkv** [comment]
Sure, @adamfils , release a link to your models, please

**Comment by larsh0103** [comment]
Any updates on pretrained models? 

Would be great to get access.

**Comment by ruchind159** [comment]
hi @adamfils, Can share the link of your trained model Please ?

**Comment by JoonHong-Kim** [comment]
Hi, @adamfils , Could you share the link of your model too? I need to use this model urgently.

**Comment by matthewkleinmann** [comment]
I too would like access to a pre trained model.  I have a slow machine with only a CPU.  This seems to be a recurring issue here.

**Comment by bballboy8** [comment]
Ditto on being very grateful for access to that pre trained model

**Comment by GeorgiyNaumenko** [comment]
Hi, @adamfils, it would be great if you gave the access to the pre-trained model :)

**Comment by bballboy8** [comment]
Yes it would be great if someone could share a pre-trained model they've had success with.

**Comment by darkdante2209** [comment]
Hi @adamfils , could you please share your pretrained model, thanks alot

**Comment by desperado1999** [comment]
Hi @adamfils could you please share your pretrained model, thanks very much

**Comment by muhammad-ahmed-ghani** [comment]
@darkdante2209 @desperado1999 Hi I trained model on open source ```librimix``` dataset and I hope if you just want the demonstration for academic research purpose than it might help you. As I don't have much resources to train, I trained it on just 31 epochs. Accuracy is still bad 0.04 train and 0.64 valid loss. It is underfitting but I don't have enough time to look at it. At least you can test it out.

[svoice_demo](https://github.com/muhammad-ahmed-ghani/svoice_demo)

**Comment by jain11vaibhav** [comment]
Hi, @adamfils , Could you please share the link of your model too? I need to do the inference for the data I have.

**Comment by desperado1999** [comment]
> 

WoW! Thanks very much! Recently I am trying to train svoice on another dataset similar to LibriMix ( the sources in each mixed file are from the same speaker), and I think your pretrianed model could significantly help me reduce my training time.

**Comment by desperado1999** [comment]
> @darkdante2209 @desperado1999 Hi I trained model on open source `librimix` dataset and I hope if you just want the demonstration for academic research purpose than it might help you. As I don't have much resources to train, it took 2 weeks for just 19 epochs on Gtx 1070. Accuracy is still bad 0.6 train loss. [svoice_demo](https://github.com/muhammad-ahmed-ghani/svoice_demo)

WoW! Thanks very much! Recently I am trying to train svoice on another dataset similar to LibriMix ( the sources in each mixed file are from the same speaker), and I think your pretrianed model could significantly help me reduce my training time.

**Comment by adamfils** [comment]
Hi Guys,
Sorry for the delayed response.
Here is a link to the model.
https://drive.google.com/file/d/1F6ll_HyyrhjWjC3J74kBdT--4a9hPMX6/view?usp=sharing

@egorsmkv @jain11vaibhav @desperado1999 @darkdante2209 @GeorgiyNaumenko @JoonHong-Kim @ruchind159

**Comment by qalabeabbas49** [comment]
hye @adamfils, is this a 2-speaker separation model? On which data did you train it?
